sensor_msgs/Image laser_image
geometry_msgs/Pose pose
geometry_msgs/PointStamped min_dist
int8 collision
sensor_msgs/LaserScan laser
geometry_msgs/Twist vel
